src/service/action-impl.js

```diff
@@ -361,9 +361,9 @@ export class ActionService {
   /**
    * @param {!Element} node
    * @param {string} actionEventType
-   * @return {?ActionInfoDef}
+   * @return {?Array<ActionInfoDef>}
    */
-  matchActionInfo_(node, actionEventType) {
+  matchActionInfos_(node, actionEventType) {
```

Comment:
The key for `actionEventType` could still be unset, which would make the result `undefined` instead of null, which is presumably the reason for the check.